src/test/web/customReporter.ts

```diff
@@ -147,32 +244,61 @@ function CustomReporter(this: any, runner: mochaTypes.Runner, options: mochaType
             });
         })
         .on(constants.EVENT_TEST_FAIL, (test: mochaTypes.Test, err: any) => {
+            const consoleOutput = consoleHijacker.release();
             writeReportProgress({
                 event: constants.EVENT_TEST_FAIL,
                 title: test.title,
                 err: formatException(err),
+                titlePath: test.titlePath(),
```

Comment:
Add some more data required by the test writers (e.g. when using xunit, json report writer or the like, these are required)